src/site/content/en/learn/learn.11tydata.js

```diff
@@ -16,6 +16,7 @@ module.exports = function () {
     allPaths['learn-core-web-vitals'],
     allPaths['metrics'],
     allPaths['fast'],
+    allPaths['how-to-optimize-inp'],
```

Comment:
Where is this used? Can't find it on the site.

Reply:
I can't find it either, so that's a good enough reason to strike it.

Reply:
Ah, okay. This is weird. It adds the content collection to the page at web.dev/explore under the performance section. It's a bit weird that the filename has `learn` in it, as it makes it seem like it would appear under the `/learn/` path. So we need to keep this in here.